apps/judicial-system/web/messages/InvestigationCases/Prosecutor/requestedHearingArrangements.ts

```diff
@@ -9,23 +9,6 @@ export const icRequestedHearingArrangements = {
       'Notaður sem titill á óskir um fyrirtöku skrefi í rannsóknarheimildum.',
   }),
   sections: {
-    prosecutor: defineMessages({
```

Comment:
All Contentful strings were moved under the new component folder ProsecutorSection.